containers/deploy/pulp/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: pulp persistent volume claim
+  k8s_raw:
+    state: "{{ deployment_state }}"
+    definition: "{{ lookup('template', 'templates/' + item + '.pvc.yaml') | from_yaml }}"
+  with_items:
+    - pulp-data
+    - puppet-data
```

Comment:
Would this PVC logically fit better in the puppet role?

Reply:
Its a tough yes and no. Just because I don't have puppet doesn't meant I can't manage puppet content and vice-versa. For example, you can have Pulp manage Puppet content act like a Puppet forge. Or you can have it put puppet modules on disk where the Puppet server expects them to be. TLDR; I don't have a good answer.

Reply:
makes sense, main reason I asked was that the puppet role will fail if this pvc has not been created, which makes this task a dependency of the puppet module.

Reply:
Or rather, the task won't fail, but the deployment will